pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -57,6 +57,13 @@ func NewClient(namespace, kubeConfigPath string) (*Client, error) {
 	}, nil
 }
 
+// NewClientFromClient creates KUDO client from kubernetes client interface
+func NewClientFromClient(client versioned.Interface) *Client {
```

Comment:
```suggestion
func NewClientFromK8S(client versioned.Interface) *Client {
```

Reply:
I don't think we can leave it "ClientFromClient".   It isn't the same client or it would be copy and it is confusing